spec/coffee/helpers/google-api-mock.coffee

```diff
@@ -1,3 +1,6 @@
+capitalize = (s) ->
```

Comment:
I didnt have much concern for this file since it is not included in dist. But valid point.